frontend/taipy-gui/src/components/Taipy/Field.tsx

```diff
@@ -75,6 +77,19 @@ const Field = (props: TaipyFieldProps) => {
                 <span className={className} id={id} style={style}>
                     {value}
                 </span>
+            ) : mode == 'latex' ? (
```

Comment:
MathJax is a JavaScript library that renders LaTeX-like mathematical expressions in web browsers. While MathJax handles the rendering, from a user perspective, they're still writing LaTeX-style math expressions. The choice to name the mode "latex" was based on the suggestion from the issue.

I hope this helps clarify the reasoning, and thank you again for your review and feedback!

Reply:
from [Wikipedia](https://en.wikipedia.org/wiki/MathJax#:~:text=MathJax%20can%20display%20mathematical%20notation,used%20to%20describe%20mathematical%20notation.): "MathJax can display mathematical notation written in [LaTeX](https://en.wikipedia.org/wiki/LaTeX) or [MathML](https://en.wikipedia.org/wiki/MathML) markup. Because MathJax is meant only for math display, whereas LaTeX is a document layout language, MathJax only supports the subset of LaTeX used to describe mathematical notation."

I would propose that we change the mode name to both `mathlatex` and `mathml` (which is supported too)
What do you think @FabienLelaquais ?

Reply:
If there's suport for Lateχ... I would keep it as is...
Documenting that oh by the way, this supports MathML as well!

But to me: "MathJax can display mathematical notation written in LaTeX..." and "MathJax only supports the subset of LaTeX..." are contradictory.

Reply:
From my understanding, I think what the Wikipedia article is trying to convey is that while MathJax can display mathematical notation written in LaTeX, it focuses specifically on rendering mathematical expressions. In contrast, LaTeX is a comprehensive document layout language that handles everything from page formatting to advanced math extensions. 

So, MathJax supports LaTeX math notation but doesn't cover the full range of LaTeX's document-related features.

Reply:
@kaze-droid, what do *you* think?

By the way, this PR is simply perfect! Well done and thank you!

Reply:
Thanks for the kind words! Personally I think it makes sense to group them under the same umbrella term of "latex" for simplicity